src/pages/VaccineAdminPage.js

```diff
@@ -54,6 +54,7 @@ const VaccineAdminPageComponent = ({
   isScanning,
   onEditLocation,
   onSaveLocation,
+  onNewLocation,
```

Comment:
was using a single callback for both editing and creating - changed to different callbacks. The callback was having a toast pop up saying "No location types sorry", when trying to edit or create. Now, just when trying to create it shows that toast